### ISSUE TITLE
Request: Larger font size

### ISSUE THREAD

**Issue by skoskie** [MiroHibler/sublime-keymaps#16]
First, thanks for the plugin. It's great!

Second, I can barely see the keymaps because of the font size. It's a little clearer on a retina display, but on a display with lower resolution, it's pretty difficult to make out the characters. Here's how it looks on a Thunderbolt display:

![screen shot 2014-02-27 at 10 36 32 am](https://f.cloud.github.com/assets/495511/2284823/aa07b370-9fcd-11e3-96d8-3a748d0f5a4d.png)

**Comment by MiroHibler** [comment]
Did you fix it?

**Comment by eightygrit** [comment]
Thanks for the follow-up. For now I've just disabled it. I've also realized this isn't a *font size* issue, it's a *font* issue. 

For some reason, other plugins display with a sans serif font, but this one displays in monospaced, which seems to be the issue. For instance, the CDNJS plugin also uses a second line of text, but look at the difference in clarity: 

![screen shot 2015-04-13 at 10 06 23 pm](https://cloud.githubusercontent.com/assets/495511/7130349/1a7fa01c-e22c-11e4-9533-12d898fa11a5.png)

And, of course, this same style is used for keymaps shown on a single line as well:

![screen shot 2015-04-13 at 10 34 24 pm](https://cloud.githubusercontent.com/assets/495511/7130449/8384f890-e22d-11e4-9e08-55402a83a91b.png)

Look at the `⌘` symbol in that screenshot vs. the original screenshot. Huge difference. 

It fully makes sense for this to be a theme issue, but I haven't made any changes to my theme recently, and I'm not having this issue with other plugins. I'm not sure what could be causing it.

**Comment by megadr01d** [comment]
@eightygrit You're right, this plugin uses a monospace font and that's the problem with the small symbols.

A quick hack is to open `Keymaps.py` from the package and on line 571 change from

```py
window.show_quick_panel(self.panel_items, self.runcommand, sublime.MONOSPACE_FONT)
```

to

```py
window.show_quick_panel(self.panel_items, self.runcommand)
```

This will use ST's default font.
